Vehicle.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal class Vehicle : IVehicle
+    {
+        // Manufacturer of the vehicle
+        public string Manufacturer { get; set; }
```

Comment:
The naming conventions for the properties should be consistent. It is recommended to follow the C# naming conventions where property names should be in PascalCase. Therefore, `model` should be renamed to `Model`, and `year` should be renamed to `Year`.